dev/conformance/runner.ts

```diff
@@ -514,17 +521,91 @@ function runTest(spec: ConformanceProto) {
   );
 }
 
+/**
+ * Parses through the TestFile and changes all instances of Timestamps encoded
+ * as strings to a proper protobuf type since protobufJS does not support it at
+ * the moment.
+ */
+function updateTimeStamp(obj: {[key: string]: {}}) {
+  const fieldNames = ['updateTime', 'createTime', 'readTime'];
+  for (const key of Object.keys(obj)) {
+    if (fieldNames.includes(key) && typeof obj[key] === 'string') {
+      obj[key] = timestamp((obj[key] as unknown) as string);
```

Comment:
Can you find a way to remove the cast here? You may have to assign obj[key] to a temporary variable.